src/components/AccountNamesTable.vue

```diff
@@ -38,6 +53,8 @@
 import AppLink from '@/components/AppLink'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 import DatetimeLabel from '@/components/DatetimeLabel'
+import { accountHints } from '~/utils/hints/accountHints'
```

Comment:
I think we should stick to one type of aliases - @ in this case

here and in other components too

Reply:
Right, fixed